src/interface/playerSelection/PlayerSelection.scss

```diff
@@ -1,87 +1,284 @@
 @import 'interface/layout/Theme.scss';
 
-.player-selection {
-  text-align: center;
-  line-height: 0;
+.player-selection-container {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
-  margin: -15px;
-
-  > a {
-    display: inline-block;
-    padding: 15px;
-    border-radius: 3px;
-    margin: 15px;
-    background: $panelColor;
-    border: 1px solid transparent;
-
-    &:hover {
-      // Placeholder
-      // TODO: Replace
-      text-decoration: none;
-      background: transparentize($primaryColor, 0.9);
-      border: 1px solid rgba(248, 183, 0, 1);
+  grid-template-columns: 1fr 0fr;
+
+  &.show-info {
+    grid-template-columns: 1fr 1fr;
+  
+    .player-selection {
+      border-right: 1px solid rgba(255, 255, 255, 0.15);
+    }
+
+    .player-info {
+      display: block;
+      padding-left: 20px;
+      margin-top: -10px;
+
+      a:hover {
+        text-decoration: none;
+      }
+
+      .player-background {
+        background-position: center;
+        background-size: cover;
+      }
+
+      .player-gear {
+        display: grid;
+        grid-template-columns: 45px 15px auto 30px 15px 45px 45px 15px 30px auto 15px 45px;
+        grid-template-rows: 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px 20px;
+        grid-template-areas:
```

Comment:
This looks very fragile and I would prefer any other solution. You could hardcode the locations, or you could make three elements (two for the columns and one for the row) and sort and group the items to let CSS auto-position them. I don't think the CSS is the best place to decide where items are rendered.

Reply:
I checked the armory, and they actually use the three elements approach.

Reply:
Fair enough :*(